packages/foundry-sdk-generator/package.json

```diff
@@ -29,24 +29,23 @@
     "transpile": "monorepo.tool.transpile tsup"
   },
   "dependencies": {
-    "@osdk/api": "workspace:~",
-    "@osdk/client": "workspace:~",
-    "@osdk/client.api": "workspace:~",
-    "@osdk/generator": "workspace:~",
-    "@osdk/internal.foundry.core": "workspace:~",
-    "@osdk/internal.foundry.ontologies": "workspace:~",
-    "@osdk/internal.foundry.ontologiesv2": "workspace:~",
-    "@osdk/legacy-client": "^2.5.0",
-    "@osdk/shared.net": "workspace:~",
-    "@rollup/plugin-commonjs": "^25.0.4",
-    "@rollup/plugin-node-resolve": "^15.2.1",
-    "consola": "^3.2.3",
-    "find-up": "^7.0.0",
-    "rollup": "^3.28.1",
-    "rollup-plugin-polyfill-node": "^0.13.0",
-    "ts-morph": "^19.0.0",
-    "typescript": "^5.5.3",
-    "yargs": "^17.7.2"
+    "@osdk/api": "workspace:*",
+    "@osdk/client": "workspace:*",
+    "@osdk/client.api": "workspace:*",
+    "@osdk/generator": "workspace:*",
+    "@osdk/internal.foundry.core": "workspace:*",
+    "@osdk/internal.foundry.ontologies": "workspace:*",
+    "@osdk/internal.foundry.ontologiesv2": "workspace:*",
+    "@osdk/shared.net": "workspace:*",
+    "@rollup/plugin-commonjs": "25.0.7",
+    "@rollup/plugin-node-resolve": "15.2.3",
+    "consola": "3.2.3",
+    "find-up": "7.0.0",
+    "rollup": "3.29.1",
+    "rollup-plugin-polyfill-node": "0.13.0",
+    "ts-morph": "19.0.0",
+    "typescript": "5.5.3",
+    "yargs": "17.7.2"
```

Comment:
By shifting all of foundry-sdk-generator to fixed packages, we should not have surprises where typescript gets upgraded underneath us and breaks our generator